lib/adapters/REST/endpoints/raw.ts

```diff
@@ -53,6 +53,7 @@ export function del<T = any>(http: AxiosInstance, url: string, config?: AxiosReq
     .then((response) => response.data, errorHandler)
 }
 
+// TODO: this is broken
```

Comment:
What is broken? Would it be part of a followup?

Reply:
Oh - forgot about that comment.
Yes, I'll fix that in a follow up PR. The whole `http` method is broken if I remember correctly.